spec/requests/collections_spec.rb

```diff
@@ -195,6 +196,18 @@
       expect(json['image']).to include "can't be blank"
     end
   end
+
+  describe "DELETE /destroy" do 
+    it 'deletes a collection' do 
+      post '/collections', params: collection_params 
```

Comment:
Where are you creating the collection? Is this test passing?

Reply:
I had to create a mock created to pass the test on my end. It was before the create branch was fully merged.